features/issues/components/issue-list/issue-list.tsx

```diff
@@ -3,6 +3,7 @@ import { ProjectLanguage } from "@api/projects.types";
 import { useGetProjects } from "@features/projects";
 import { useGetIssues } from "../../api/use-get-issues";
 import { IssueRow } from "./issue-row";
+import { LoadingSpinner } from "../../../ui/loading-spinner";
```

Comment:
This could be imported via `@features/ui`

Reply:
Thanks for that tip!